python/rikai/contrib/torchhub/pytorch/vision/__init__.py

```diff
@@ -0,0 +1,21 @@
+#  Copyright 2021 Rikai Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from . import resnet
```

Comment:
does this only work with touchhub? what about the officially blessed models?

https://pytorch.org/vision/stable/models.html


Also i think `torchhub` will only have `pytorch` models, right? why do we need `torchhub/pytorch` modules?

Reply:
because here the convention is `torchhub/{org_name}/{project_name}`